server_addon/maya/server/settings/publishers.py

```diff
@@ -309,6 +386,166 @@ class ExtractAlembicModel(BaseSettingsModel):
         description="List of attribute prefixes for attributes that will be "
                     "included in the alembic export.",
     )
+    dataFormat: str = SettingsField(
+        enum_resolver=extract_alembic_data_format_enum,
+        title="Data Format",
+        description="The data format to use to write the file."
+    )
+    dontSkipUnwrittenFrames: bool = SettingsField(
+        title="Dont Skip Unwritten Frames",
+        description=(
+            "When evaluating multiple translate jobs, this decides whether to "
+            "evaluate frames between jobs when there is a gap in their frame "
+            "ranges."
+        )
+    )
+    eulerFilter: bool = SettingsField(
+        title="Euler Filter",
+        description="Apply Euler filter while sampling rotations."
+    )
+    melPerFrameCallback: str = SettingsField(
+        title="Mel Per Frame Callback",
+        description=(
+            "When each frame (and the static frame) is evaluated the string "
+            "specified is evaluated as a Mel command."
+        )
+    )
+    melPostFrameCallback: str = SettingsField(
+        title="Mel Post Frame Callback",
+        description=(
+            "When the translation has finished the string specified is "
+            "evaluated as a Mel command."
+        )
+    )
+    noNormals: bool = SettingsField(
+        title="No Normals",
+        description=(
+            "Present normal data for Alembic poly meshes will not be written."
+        )
+    )
+    preRoll: bool = SettingsField(
+        title="Pre Roll",
+        description=(
+            "When enabled, the pre roll start frame is used to pre roll the "
+            "export for."
```

Comment:
Can we add in the description whether that 'pre roll' is included in the exported data or not. I'm a bit unsure myself. Is it just starting from there for simulation purposes for the export, or is it included in the exported data?

Reply:
Suggested an expand description.